inheritance/lib/src/main/java/inheritance/Review.java

```diff
@@ -62,7 +62,6 @@ private void updateStarRating () {
         }
     }
 
-
```

Comment:
Because this Review class is in the same package as the Restaurant class, you don't need to recreate the Restaurant class here. The declaration that you already have on line 10 will bring the Restaurant in correctly.